### PR TITLE
BoH bombing rework port

### PR DESCRIPTION
**Pull request by SomeAngryMiner**
##About The Pull Request

One of the last /tg/ port PRs i had in mind.
 this one reworks the bag of holding bomb into something more interesting, and adds a button for admins for damage control, reverting its effects in case of griefing.
Ancient ones used to simply spawn a singularity, the ones we used to have  spawned a tear in reality, while these new ones spawn a stationary singulo with an extreme gravitational pull, yoinking everything around it inside, and deleting them if an admin doesnt revert it.

All code made by Nicbn, the original coder of this feature.

## Why It's Good For The Game

I believe this makes the BoH bombing fairly more interesting and less dull, besides being an admin QoL change by having the ability to revert it in case of a griefer.

## Changelog
:cl:
add: BoH bombing reworked! instead of a chasm, spawns a stationary singulo with an absurd pull, ready to eat everything inside
admin: To pair the rework, admins get a button to revert any changes made by the BoH bombing that expires after 10 minutes, perfect for damage control in case of griefers.
/:cl:

### CODE REVIEW

**Comment by SomeAngryMiner** [comment]
odd, Travis says that it detects pixel_x/pixel_y and something about check_grep, i  dont know what is causing the check errors.

**Comment by ike709** [comment]
For some reason you edited the centcom map, and also didn't mapmerge it after.

File: beestation.dme [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 // BEGIN_INCLUDE
 #include "_maps\_basemap.dm"
+#include "_maps\map_files\generic\CentCom.dmm"
```
Comment (qwertyquerty) [inline beestation.dme:17]: why

Comment (SomeAngryMiner) [reply beestation.dme:17]: i fucked up some stuff, ill fix it.

**Pull request closed without merging**